### ISSUE TITLE
Changing Duration Out does not have any effect on bounceIn/bounceOut animation

### ISSUE THREAD

**Issue by pravin-d** [nearform/react-animation#22]
I have a react component which has text changing. Changing durationOut does not affect the actual visible animation speed. Any help would be appreciated 
```
const Component = props => {
const numberWithCommas = number => number;
return (
 <div className="col-md-6 offset-md-3 text-center">
    <label>Some Label</label>
    <span className="btn btn-outline-primary-nohover w-100">
      <AnimateOnChange
        animationIn="bounceIn"
        animationOut="bounceOut"
        durationOut={500}
      >
        {numberWithCommas(props.somenumber)}
      </AnimateOnChange>
    </span>
  </div>
)
```

**Comment by AlexOros** [comment]
Hmm I downgraded to version 1.0.9 and it works now.

**Comment by donovanh** [comment]
Thanks @pravin-d and @AlexOros - I've issued a PR that should address this issue. I'll update and close this issue once it's merged.

https://github.com/nearform/react-animation/pull/24

**Comment by donovanh** [comment]
This should now be fixed in version `1.2.2` - sorry about that!
